outputs.tf

```
@@ -2,8 +2,8 @@ output "fargate_service" {
   value = aws_ecs_service.service
 }
 
-output "ecs_cluster" {
-  value = local.create_new_cluster ? aws_ecs_cluster.new_cluster[0] : data.aws_ecs_cluster.existing_cluster[0]
+output "new_ecs_cluster" {
+  value = local.create_new_cluster ? aws_ecs_cluster.new_cluster[0] : var.existing_ecs_cluster.arn
```

I think we wanted to return null if the existing cluster was defined
```suggestion
  value = local.create_new_cluster ? aws_ecs_cluster.new_cluster[0] : null
```